extra/s3-uploads.sh

```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/env nix-shell
+#! nix-shell -i bash -p minio-client zstd
+
+DEV_SHELLS=(
+    "ghc8107"
+    "ghc902"
+    "ghc925"
+    "ghc8107-minimal"
+    "ghc902-minimal"
+    "ghc925-minimal"
+    "ghc8107-static-minimal"
+    "ghc902-static-minimal"
+    "ghc925-static-minimal"
+)
+
+SYSTEMS=("aarch64-darwin" "aarch64-linux" "x86_64-darwin" "x86_64-linux")
+
+mc alias set zw3rk https://s3.zw3rk.com "$ACCESSKEY" "$SECRETKEY"
```

Comment:
Let's see if there are any generic s3 support items for GH? Then we don't need mc :D